system-parachains/encointer/src/lib.rs

```diff
@@ -545,6 +545,29 @@ impl pallet_encointer_faucet::Config for Runtime {
 	type WeightInfo = weights::pallet_encointer_faucet::WeightInfo<Runtime>;
 }
 
+parameter_types! {
+	pub const ConfirmationPeriod: Moment = 2 * 24 * 3600 * 1000; // [ms]
+	pub const ProposalLifetime: Moment = 9 * 24 * 3600 * 1000; // [ms]
+}
+
+impl pallet_encointer_democracy::Config for Runtime {
+	type RuntimeEvent = RuntimeEvent;
+	type MaxReputationCount = ConstU32<64>;
+	type ConfirmationPeriod = ConfirmationPeriod;
+	type ProposalLifetime = ProposalLifetime;
+	type MinTurnout = ConstU128<50>; // permill of electorate: 1 = 0.1%, 50 = 5.0%
```

Comment:
Wouldn't 5% of a permill be `50_000`?

Reply:
good catch. actually, the confusion lies deeper. this setting is correct as-is according to the pallet implementation which treats the value as perthousand. I'll adjust the naming:
https://github.com/encointer/pallets/blob/43e099dce05f44ce3d4d945117b43099d5718aa1/democracy/src/lib.rs#L559-L562